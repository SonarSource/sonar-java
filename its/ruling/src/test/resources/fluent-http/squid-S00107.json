{
'net.code-story:http:src/main/java/net/codestory/http/misc/Env.java':[
54,
],
'net.code-story:http:src/main/java/net/codestory/http/templating/Model.java':[
51,
],
'net.code-story:http:src/main/java/net/codestory/http/templating/ModelAndView.java':[
47,
],
}
