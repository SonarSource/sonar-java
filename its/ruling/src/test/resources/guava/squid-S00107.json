{
'com.google.guava:guava:src/com/google/common/cache/LocalCache.java':[
4606,
],
'com.google.guava:guava:src/com/google/common/collect/ComputingConcurrentHashMap.java':[
403,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableBiMap.java':[
75,
85,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableList.java':[
138,
147,
156,
166,
180,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableListMultimap.java':[
88,
101,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMap.java':[
95,
105,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMultimap.java':[
105,
114,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSetMultimap.java':[
102,
117,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSortedMapFauxverideShim.java':[
95,
110,
],
'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
3917,
3990,
],
'com.google.guava:guava:src/com/google/common/primitives/Longs.java':[
305,
],
}
