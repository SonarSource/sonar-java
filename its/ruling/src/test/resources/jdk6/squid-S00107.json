{
'jdk6:java/awt/Event.java':[
611,
],
'jdk6:java/awt/FlowLayout.java':[
499,
],
'jdk6:java/awt/GradientPaintContext.java':[
231,
249,
],
'jdk6:java/awt/Graphics.java':[
1046,
1107,
],
'jdk6:java/awt/GridBagConstraints.java':[
610,
],
'jdk6:java/awt/JobAttributes.java':[
318,
],
'jdk6:java/awt/LinearGradientPaintContext.java':[
59,
],
'jdk6:java/awt/MultipleGradientPaintContext.java':[
132,
],
'jdk6:java/awt/RadialGradientPaint.java':[
321,
413,
],
'jdk6:java/awt/RadialGradientPaintContext.java':[
91,
],
'jdk6:java/awt/TexturePaintContext.java':[
293,
],
'jdk6:java/awt/event/KeyEvent.java':[
936,
],
'jdk6:java/awt/event/MouseEvent.java':[
446,
496,
544,
],
'jdk6:java/awt/event/MouseWheelEvent.java':[
161,
208,
],
'jdk6:java/awt/font/GlyphJustificationInfo.java':[
84,
],
'jdk6:java/awt/font/TextLine.java':[
85,
968,
],
'jdk6:java/awt/geom/CubicCurve2D.java':[
131,
276,
414,
694,
787,
821,
1298,
],
'jdk6:java/awt/geom/Line2D.java':[
586,
],
'jdk6:java/awt/geom/QuadCurve2D.java':[
1039,
],
'jdk6:java/awt/image/AreaAveragingScaleFilter.java':[
117,
],
'jdk6:java/awt/image/DirectColorModel.java':[
213,
],
'jdk6:java/awt/image/ImageConsumer.java':[
142,
168,
],
'jdk6:java/awt/image/PackedColorModel.java':[
176,
],
'jdk6:java/awt/image/PixelGrabber.java':[
101,
130,
],
'jdk6:java/lang/ClassLoader.java':[
502,
1442,
],
'jdk6:java/lang/Package.java':[
405,
],
'jdk6:java/lang/UNIXProcess.java':[
35,
],
'jdk6:java/lang/management/ThreadInfo.java':[
108,
135,
198,
],
'jdk6:java/lang/reflect/Constructor.java':[
101,
],
'jdk6:java/lang/reflect/Method.java':[
105,
],
'jdk6:java/lang/reflect/ReflectAccess.java':[
35,
60,
],
'jdk6:java/net/Authenticator.java':[
269,
],
'jdk6:java/net/URI.java':[
1841,
1889,
],
'jdk6:java/net/URL.java':[
680,
],
'jdk6:java/net/URLStreamHandler.java':[
513,
],
'jdk6:java/nio/Bits.java':[
285,
],
'jdk6:java/security/spec/RSAMultiPrimePrivateCrtKeySpec.java':[
72,
],
'jdk6:java/security/spec/RSAPrivateCrtKeySpec.java':[
56,
],
'jdk6:java/text/DecimalFormat.java':[
845,
],
'jdk6:java/text/RBCollationTables.java':[
94,
],
'jdk6:java/text/SimpleDateFormat.java':[
1548,
],
'jdk6:java/util/SimpleTimeZone.java':[
197,
255,
311,
728,
],
}
