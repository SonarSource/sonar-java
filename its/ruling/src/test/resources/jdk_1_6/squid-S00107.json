{
'jdk_1_6:java/awt/Event.java':[
611,
],
'jdk_1_6:java/awt/FlowLayout.java':[
499,
],
'jdk_1_6:java/awt/GradientPaintContext.java':[
231,
249,
],
'jdk_1_6:java/awt/Graphics.java':[
1046,
1107,
],
'jdk_1_6:java/awt/GridBagConstraints.java':[
610,
],
'jdk_1_6:java/awt/JobAttributes.java':[
318,
],
'jdk_1_6:java/awt/LinearGradientPaintContext.java':[
59,
],
'jdk_1_6:java/awt/MultipleGradientPaintContext.java':[
132,
],
'jdk_1_6:java/awt/RadialGradientPaint.java':[
321,
413,
],
'jdk_1_6:java/awt/RadialGradientPaintContext.java':[
91,
],
'jdk_1_6:java/awt/TexturePaintContext.java':[
293,
],
'jdk_1_6:java/awt/event/KeyEvent.java':[
936,
],
'jdk_1_6:java/awt/event/MouseEvent.java':[
446,
496,
544,
],
'jdk_1_6:java/awt/event/MouseWheelEvent.java':[
161,
208,
],
'jdk_1_6:java/awt/font/GlyphJustificationInfo.java':[
84,
],
'jdk_1_6:java/awt/font/TextLine.java':[
85,
968,
],
'jdk_1_6:java/awt/geom/CubicCurve2D.java':[
131,
276,
414,
694,
787,
821,
1298,
],
'jdk_1_6:java/awt/geom/Line2D.java':[
586,
],
'jdk_1_6:java/awt/geom/QuadCurve2D.java':[
1039,
],
'jdk_1_6:java/awt/image/AreaAveragingScaleFilter.java':[
117,
],
'jdk_1_6:java/awt/image/DirectColorModel.java':[
213,
],
'jdk_1_6:java/awt/image/ImageConsumer.java':[
142,
168,
],
'jdk_1_6:java/awt/image/PackedColorModel.java':[
176,
],
'jdk_1_6:java/awt/image/PixelGrabber.java':[
101,
130,
],
'jdk_1_6:java/lang/ClassLoader.java':[
502,
1442,
],
'jdk_1_6:java/lang/Package.java':[
405,
],
'jdk_1_6:java/lang/UNIXProcess.java':[
35,
],
'jdk_1_6:java/lang/management/ThreadInfo.java':[
108,
135,
198,
],
'jdk_1_6:java/lang/reflect/Constructor.java':[
101,
],
'jdk_1_6:java/lang/reflect/Method.java':[
105,
],
'jdk_1_6:java/net/Authenticator.java':[
269,
],
'jdk_1_6:java/net/URI.java':[
1841,
1889,
],
'jdk_1_6:java/net/URL.java':[
680,
],
'jdk_1_6:java/net/URLStreamHandler.java':[
513,
],
'jdk_1_6:java/nio/Bits.java':[
285,
],
'jdk_1_6:java/security/spec/RSAMultiPrimePrivateCrtKeySpec.java':[
72,
],
'jdk_1_6:java/security/spec/RSAPrivateCrtKeySpec.java':[
56,
],
'jdk_1_6:java/text/DecimalFormat.java':[
845,
],
'jdk_1_6:java/text/RBCollationTables.java':[
94,
],
'jdk_1_6:java/text/SimpleDateFormat.java':[
1548,
],
'jdk_1_6:java/util/SimpleTimeZone.java':[
197,
255,
311,
728,
],
}
