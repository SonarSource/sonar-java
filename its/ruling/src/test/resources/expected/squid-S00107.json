{
'project:com/sun/corba/se/impl/corba/RequestImpl.java':[
102,
],
'project:com/sun/corba/se/impl/protocol/CorbaMessageMediatorImpl.java':[
171,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/MessageBase.java':[
450,
503,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/RequestMessage_1_1.java':[
67,
],
'project:com/sun/imageio/plugins/bmp/BMPImageReader.java':[
1679,
1695,
1709,
1716,
],
'project:com/sun/imageio/plugins/common/ReaderUtil.java':[
43,
175,
],
'project:com/sun/imageio/plugins/gif/GIFImageWriter.java':[
817,
849,
1057,
1247,
],
'project:com/sun/imageio/plugins/jpeg/JPEGImageReader.java':[
1290,
],
'project:com/sun/imageio/plugins/jpeg/JPEGImageWriter.java':[
1627,
],
'project:com/sun/imageio/plugins/jpeg/JPEGMetadata.java':[
2333,
],
'project:com/sun/java/swing/plaf/gtk/GTKEngine.java':[
121,
124,
127,
134,
137,
140,
146,
155,
158,
378,
388,
401,
412,
420,
430,
440,
451,
459,
469,
477,
485,
498,
508,
516,
],
'project:com/sun/java/swing/plaf/gtk/GTKPainter.java':[
285,
1190,
1202,
1245,
1306,
1344,
],
'project:com/sun/java/swing/plaf/gtk/Metacity.java':[
1427,
],
'project:com/sun/java/swing/plaf/motif/MotifBorders.java':[
567,
],
'project:com/sun/java/swing/plaf/motif/MotifGraphicsUtils.java':[
99,
339,
],
'project:com/sun/java/swing/plaf/motif/MotifIconFactory.java':[
194,
],
'project:com/sun/java/swing/plaf/motif/MotifTabbedPaneUI.java':[
157,
183,
],
'project:com/sun/java/swing/plaf/windows/AnimationController.java':[
235,
],
'project:com/sun/java/swing/plaf/windows/WindowsGraphicsUtils.java':[
125,
],
'project:com/sun/java/swing/plaf/windows/WindowsInternalFrameTitlePane.java':[
395,
],
'project:com/sun/java/swing/plaf/windows/WindowsTabbedPaneUI.java':[
157,
164,
201,
],
'project:com/sun/jmx/snmp/agent/SnmpMibAgent.java':[
581,
],
'project:com/sun/jmx/snmp/agent/SnmpMibRequestImpl.java':[
61,
],
'project:com/sun/jmx/snmp/agent/SnmpMibTable.java':[
2066,
],
'project:com/sun/jmx/snmp/daemon/SnmpAdaptorServer.java':[
1285,
1579,
1640,
],
'project:com/sun/jmx/snmp/daemon/SnmpRequestHandler.java':[
95,
],
'project:com/sun/jmx/snmp/daemon/SnmpSubBulkRequestHandler.java':[
62,
],
'project:com/sun/jmx/snmp/internal/SnmpMsgProcessingModel.java':[
88,
112,
],
'project:com/sun/jmx/snmp/internal/SnmpMsgProcessingSubSystem.java':[
106,
132,
],
'project:com/sun/jmx/snmp/internal/SnmpSecurityModel.java':[
56,
88,
121,
152,
],
'project:com/sun/jmx/snmp/internal/SnmpSecuritySubSystem.java':[
70,
101,
134,
165,
],
'project:com/sun/org/apache/bcel/internal/classfile/Code.java':[
156,
],
'project:com/sun/org/apache/bcel/internal/classfile/JavaClass.java':[
133,
220,
],
'project:com/sun/org/apache/bcel/internal/generic/MethodGen.java':[
117,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/CompareGenerator.java':[
58,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/MatchGenerator.java':[
47,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/MethodGenerator.java':[
94,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/NamedMethodGenerator.java':[
45,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/RtMethodGenerator.java':[
44,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/TestGenerator.java':[
53,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
724,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/XSLTCDTMManager.java':[
233,
272,
],
'project:com/sun/org/apache/xerces/internal/dom/events/MutationEventImpl.java':[
100,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/DTDGrammar.java':[
604,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDProcessor.java':[
869,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLEntityDecl.java':[
130,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLSimpleType.java':[
154,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/AbstractDateTimeDV.java':[
941,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
323,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/SchemaGrammar.java':[
1187,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSAttributeDecl.java':[
74,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSComplexTypeDecl.java':[
101,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
1218,
1251,
1298,
1350,
1392,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/opti/DefaultXMLDocumentHandler.java':[
540,
],
'project:com/sun/org/apache/xerces/internal/jaxp/UnparsedEntityHandler.java':[
136,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/DatatypeFactoryImpl.java':[
631,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/XMLGregorianCalendarImpl.java':[
472,
543,
689,
767,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractDOMParser.java':[
2371,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractSAXParser.java':[
920,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractXMLDocumentParser.java':[
507,
],
'project:com/sun/org/apache/xerces/internal/parsers/DOMParserImpl.java':[
1284,
],
'project:com/sun/org/apache/xerces/internal/parsers/DTDParser.java':[
221,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
1142,
],
'project:com/sun/org/apache/xerces/internal/xni/XMLDTDHandler.java':[
238,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBase.java':[
192,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseIterators.java':[
78,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseTraversers.java':[
83,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM.java':[
246,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM2.java':[
1860,
],
'project:java/awt/Event.java':[
628,
],
'project:java/awt/FlowLayout.java':[
514,
],
'project:java/awt/GradientPaintContext.java':[
250,
268,
],
'project:java/awt/Graphics.java':[
1063,
1124,
],
'project:java/awt/GridBagConstraints.java':[
627,
],
'project:java/awt/JobAttributes.java':[
335,
],
'project:java/awt/LinearGradientPaintContext.java':[
77,
],
'project:java/awt/MultipleGradientPaintContext.java':[
150,
],
'project:java/awt/RadialGradientPaint.java':[
358,
451,
],
'project:java/awt/RadialGradientPaintContext.java':[
109,
],
'project:java/awt/TexturePaintContext.java':[
314,
403,
541,
664,
754,
],
'project:java/awt/event/KeyEvent.java':[
925,
],
'project:java/awt/event/MouseEvent.java':[
550,
616,
729,
],
'project:java/awt/event/MouseWheelEvent.java':[
195,
242,
296,
],
'project:java/awt/font/GlyphJustificationInfo.java':[
102,
],
'project:java/awt/font/TextLine.java':[
103,
997,
],
'project:java/awt/geom/CubicCurve2D.java':[
152,
260,
297,
435,
543,
715,
808,
842,
],
'project:java/awt/geom/Line2D.java':[
603,
],
'project:java/awt/geom/QuadCurve2D.java':[
1056,
],
'project:java/awt/image/AreaAveragingScaleFilter.java':[
134,
246,
272,
],
'project:java/awt/image/BufferedImageFilter.java':[
161,
248,
],
'project:java/awt/image/CropImageFilter.java':[
115,
155,
],
'project:java/awt/image/DirectColorModel.java':[
230,
],
'project:java/awt/image/ImageConsumer.java':[
159,
185,
],
'project:java/awt/image/ImageFilter.java':[
158,
175,
],
'project:java/awt/image/PackedColorModel.java':[
193,
],
'project:java/awt/image/PixelGrabber.java':[
118,
147,
471,
558,
],
'project:java/awt/image/RGBImageFilter.java':[
224,
261,
],
'project:java/awt/image/ReplicateScaleFilter.java':[
186,
227,
],
'project:java/lang/ClassLoader.java':[
1592,
],
'project:java/lang/Package.java':[
423,
],
'project:java/lang/UNIXProcess.java':[
83,
127,
],
'project:java/lang/invoke/FilterGeneric.java':[
882,
885,
888,
891,
894,
897,
900,
903,
906,
910,
914,
918,
922,
926,
930,
934,
938,
942,
945,
948,
951,
954,
957,
960,
963,
966,
969,
981,
985,
989,
993,
997,
1001,
1005,
1009,
1013,
1017,
1022,
1027,
1032,
1037,
1042,
1047,
1052,
1057,
1062,
1067,
1071,
1075,
1079,
1083,
1087,
1091,
1095,
1099,
1103,
1107,
1120,
1124,
1128,
1132,
1136,
1140,
1144,
1148,
1152,
1156,
1160,
1165,
1170,
1175,
1180,
1185,
1190,
1195,
1200,
1205,
1210,
1215,
1219,
1223,
1227,
1231,
1235,
1239,
1243,
1247,
1251,
1255,
1259,
1272,
1277,
1282,
1287,
1292,
1297,
1302,
1307,
1312,
1317,
1322,
1327,
1332,
1337,
1342,
1347,
1352,
1357,
1362,
1367,
1372,
1377,
1382,
1387,
1392,
1397,
1402,
1407,
1412,
1417,
1422,
1427,
1432,
1437,
1441,
1445,
1458,
1463,
1468,
1473,
1478,
1483,
1488,
1493,
1498,
1503,
1508,
1513,
1518,
1523,
1528,
1533,
1538,
1543,
1548,
1553,
1558,
1563,
1568,
1573,
1579,
1584,
1589,
1594,
1599,
1604,
1609,
1614,
1619,
1624,
1629,
1634,
1638,
1643,
1647,
1660,
1666,
1672,
1678,
1684,
1690,
1696,
1702,
1708,
1714,
1720,
1726,
1732,
1738,
1744,
1750,
1756,
1762,
1768,
1774,
1780,
1786,
1792,
1798,
1804,
1811,
1818,
1824,
1830,
1836,
1842,
1848,
1854,
1860,
1866,
1872,
1878,
1884,
1889,
1895,
1901,
1906,
1920,
1926,
1932,
1938,
1944,
1950,
1956,
1962,
1968,
1974,
1980,
1986,
1992,
1998,
2004,
2010,
2016,
2022,
2028,
2034,
2040,
2046,
2052,
2058,
2064,
2070,
2077,
2084,
2091,
2097,
2103,
2109,
2115,
2121,
2127,
2133,
2139,
2145,
2151,
2157,
2162,
2168,
2174,
2180,
2185,
2199,
2205,
2211,
2217,
2223,
2229,
2235,
2241,
2247,
2253,
2259,
2265,
2271,
2277,
2283,
2289,
2295,
2301,
2307,
2313,
2319,
2325,
2331,
2337,
2343,
2349,
2355,
2362,
2369,
2376,
2383,
2389,
2395,
2401,
2407,
2413,
2419,
2425,
2431,
2437,
2443,
2449,
2454,
2460,
2466,
2472,
2478,
2483,
2497,
2503,
2509,
2515,
2521,
2527,
2533,
2539,
2545,
2551,
2557,
2563,
2569,
2575,
2581,
2587,
2593,
2599,
2605,
2611,
2617,
2623,
2629,
2635,
2641,
2647,
2653,
2659,
2666,
2673,
2680,
2687,
2694,
2700,
2706,
2712,
2718,
2724,
2730,
2736,
2742,
2748,
2754,
2760,
2765,
2771,
2777,
2783,
2789,
2795,
2800,
2814,
2821,
2828,
2835,
2842,
2849,
2856,
2863,
2870,
2877,
2884,
2891,
2898,
2905,
2912,
2919,
2926,
2933,
2940,
2947,
2954,
2961,
2968,
2975,
2982,
2989,
2996,
3003,
3010,
3018,
3026,
3034,
3042,
3050,
3058,
3065,
3072,
3079,
3086,
3093,
3100,
3107,
3114,
3121,
3128,
3135,
3141,
3148,
3155,
3162,
3169,
3176,
3183,
3189,
3204,
3211,
3218,
3225,
3232,
3239,
3246,
3253,
3260,
3267,
3274,
3281,
3288,
3295,
3302,
3309,
3316,
3323,
3330,
3337,
3344,
3351,
3358,
3365,
3372,
3379,
3386,
3393,
3400,
3407,
3415,
3423,
3431,
3439,
3447,
3455,
3463,
3470,
3477,
3484,
3491,
3498,
3505,
3512,
3519,
3526,
3533,
3540,
3546,
3553,
3560,
3567,
3574,
3581,
3588,
3595,
3601,
3616,
3623,
3630,
3637,
3644,
3651,
3658,
3665,
3672,
3679,
3686,
3693,
3700,
3707,
3714,
3721,
3728,
3735,
3742,
3749,
3756,
3763,
3770,
3777,
3784,
3791,
3798,
3805,
3812,
3819,
3826,
3834,
3842,
3850,
3858,
3866,
3874,
3882,
3890,
3897,
3904,
3911,
3918,
3925,
3932,
3939,
3946,
3953,
3960,
3967,
3973,
3980,
3987,
3994,
4001,
4008,
4015,
4022,
4029,
4035,
4050,
4057,
4064,
4071,
4078,
4085,
4092,
4099,
4106,
4113,
4120,
4127,
4134,
4141,
4148,
4155,
4162,
4169,
4176,
4183,
4190,
4197,
4204,
4211,
4218,
4225,
4232,
4239,
4246,
4253,
4260,
4267,
4275,
4283,
4291,
4299,
4307,
4315,
4323,
4331,
4339,
4346,
4353,
4360,
4367,
4374,
4381,
4388,
4395,
4402,
4409,
4416,
4422,
4429,
4436,
4443,
4450,
4457,
4464,
4471,
4478,
4485,
4491,
],
'project:java/lang/invoke/FromGeneric.java':[
603,
604,
605,
606,
607,
615,
616,
617,
618,
619,
627,
628,
629,
630,
631,
],
'project:java/lang/invoke/MethodHandleImpl.java':[
225,
1003,
1176,
],
'project:java/lang/invoke/SpreadGeneric.java':[
535,
566,
568,
601,
603,
606,
641,
643,
646,
649,
],
'project:java/lang/invoke/ToGeneric.java':[
932,
933,
934,
935,
936,
937,
938,
939,
940,
941,
942,
943,
944,
945,
946,
947,
948,
949,
950,
951,
952,
953,
954,
955,
956,
957,
958,
959,
960,
961,
962,
963,
964,
965,
966,
967,
968,
974,
975,
976,
977,
978,
979,
980,
981,
982,
983,
984,
985,
986,
987,
988,
989,
990,
991,
992,
993,
994,
995,
996,
997,
998,
999,
1000,
1001,
1002,
1003,
1004,
1005,
1006,
1007,
1008,
1009,
1010,
1011,
1012,
1013,
1014,
1020,
1021,
1022,
1023,
1024,
1025,
1026,
1027,
1028,
1029,
1030,
1031,
1032,
1033,
1034,
1035,
1036,
1037,
1038,
1039,
1040,
1041,
1042,
1043,
1044,
1045,
1046,
1047,
1048,
1049,
1050,
1051,
1052,
1053,
1054,
1055,
1056,
1057,
1058,
1059,
1060,
1061,
1062,
1063,
1064,
],
'project:java/lang/management/ThreadInfo.java':[
126,
153,
216,
],
'project:java/lang/reflect/Constructor.java':[
107,
],
'project:java/lang/reflect/Method.java':[
112,
],
'project:java/lang/reflect/ReflectAccess.java':[
53,
78,
],
'project:java/net/Authenticator.java':[
286,
],
'project:java/net/URI.java':[
1859,
1907,
],
'project:java/net/URL.java':[
698,
],
'project:java/net/URLStreamHandler.java':[
530,
],
'project:java/nio/Bits.java':[
299,
],
'project:java/security/spec/RSAMultiPrimePrivateCrtKeySpec.java':[
89,
],
'project:java/security/spec/RSAPrivateCrtKeySpec.java':[
73,
],
'project:java/text/DecimalFormat.java':[
864,
],
'project:java/text/RBCollationTables.java':[
112,
],
'project:java/text/SimpleDateFormat.java':[
1749,
],
'project:java/util/SimpleTimeZone.java':[
214,
272,
328,
745,
],
'project:javax/imageio/ImageReader.java':[
2199,
2241,
2302,
2345,
],
'project:javax/imageio/event/IIOReadUpdateListener.java':[
76,
145,
195,
227,
],
'project:javax/imageio/metadata/IIOMetadataFormatImpl.java':[
572,
],
'project:javax/imageio/spi/ImageReaderSpi.java':[
194,
],
'project:javax/imageio/spi/ImageReaderWriterSpi.java':[
215,
],
'project:javax/imageio/spi/ImageWriterSpi.java':[
195,
],
'project:javax/management/AttributeChangeNotification.java':[
99,
],
'project:javax/management/monitor/MonitorNotification.java':[
204,
],
'project:javax/management/openmbean/OpenMBeanAttributeInfoSupport.java':[
310,
382,
396,
],
'project:javax/management/relation/RelationNotification.java':[
246,
300,
466,
],
'project:javax/security/auth/kerberos/KerberosTicket.java':[
229,
245,
264,
],
'project:javax/sound/sampled/AudioFormat.java':[
218,
],
'project:javax/sound/sampled/FloatControl.java':[
139,
],
'project:javax/swing/BufferStrategyPaintManager.java':[
314,
],
'project:javax/swing/CellRendererPane.java':[
122,
],
'project:javax/swing/DebugGraphics.java':[
1181,
1231,
],
'project:javax/swing/JOptionPane.java':[
855,
1274,
],
'project:javax/swing/JViewport.java':[
1567,
],
'project:javax/swing/RepaintManager.java':[
1219,
1433,
],
'project:javax/swing/SizeRequirements.java':[
323,
357,
],
'project:javax/swing/SwingUtilities.java':[
841,
910,
941,
],
'project:javax/swing/border/MatteBorder.java':[
147,
],
'project:javax/swing/event/MenuDragMouseEvent.java':[
78,
116,
],
'project:javax/swing/event/MenuKeyEvent.java':[
71,
],
'project:javax/swing/plaf/basic/BasicGraphicsUtils.java':[
53,
123,
179,
],
'project:javax/swing/plaf/basic/BasicProgressBarUI.java':[
751,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
1013,
1054,
1177,
1237,
],
'project:javax/swing/plaf/basic/BasicTreeUI.java':[
1346,
1477,
1516,
],
'project:javax/swing/plaf/metal/MetalTabbedPaneUI.java':[
96,
119,
352,
469,
626,
727,
],
'project:javax/swing/plaf/metal/MetalTreeUI.java':[
215,
],
'project:javax/swing/plaf/metal/MetalUtils.java':[
211,
253,
298,
321,
347,
],
'project:javax/swing/plaf/nimbus/ImageCache.java':[
199,
],
'project:javax/swing/plaf/nimbus/ImageScalingHelper.java':[
102,
228,
267,
],
'project:javax/swing/plaf/nimbus/NimbusDefaults.java':[
1299,
1589,
],
'project:javax/swing/plaf/nimbus/SynthPainterImpl.java':[
52,
2097,
2157,
],
'project:javax/swing/plaf/synth/ImagePainter.java':[
780,
792,
],
'project:javax/swing/plaf/synth/ParsedSynthStyle.java':[
1045,
1065,
1943,
1957,
],
'project:javax/swing/plaf/synth/SynthGraphicsUtils.java':[
99,
141,
191,
222,
267,
362,
427,
487,
],
'project:javax/swing/plaf/synth/SynthMenuItemLayoutHelper.java':[
107,
],
'project:javax/swing/plaf/synth/SynthPainter.java':[
1776,
1817,
],
'project:javax/swing/plaf/synth/SynthTabbedPaneUI.java':[
656,
683,
],
'project:javax/swing/plaf/synth/SynthTreeUI.java':[
476,
521,
590,
],
'project:javax/swing/text/Utilities.java':[
116,
306,
329,
],
'project:javax/swing/text/html/HTMLDocument.java':[
1644,
],
'project:javax/swing/text/html/StyleSheet.java':[
1304,
2303,
],
'project:javax/swing/text/html/parser/DTD.java':[
205,
258,
],
'project:javax/xml/datatype/DatatypeFactory.java':[
823,
859,
],
'project:javax/xml/ws/spi/Provider.java':[
445,
],
'project:org/w3c/dom/events/MouseEvent.java':[
169,
],
'project:org/w3c/dom/events/MutationEvent.java':[
128,
],
}
